libnmstate/nm/connection.py

```diff
@@ -25,6 +25,9 @@
 from .active_connection import ActiveConnection
 
 
+NM_MANAGER_ERROR_DOMAIN = "nm-manager-error-quark"
```

Comment:
It also appears in `active_connection.py`, perhaps move it to the `nmclient` module.

Reply:
Done, thanks!